sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -76,31 +76,155 @@ Sample code snippets are provided to illustrate using long-running operations [b
 
 ## Key concepts
 
-Radiology Insights currently supports one document from one patient. Please take a look [here][inferences] for more detailed information about the inferences this service produces. 
+Once you've initialized a 'RadiologyInsightsClient', you can use it to analyse document text by displaying inferences found within the text.
+* Age Mismatch
+* Laterality Discrepancy
+* Sex Mismatch
+* Complete Order Discrepancy
+* Limited Order Discrepancy
+* Finding
+* Critical Result
+* Follow-up Recommendation
+* Communication
+* Radiology Procedure
 
-## Examples
+Radiology Insights currently supports one document from one patient. Please take a look [here][inferences] for more detailed information about the inferences this service produces.
 
-### Create a RadiologyInsights request and get the result using an asynchronous client
+## Examples
 
+For each inference samples are available that show how to retrieve the information either in a synchronous (block until operation is complete, slower) or in an asynchronous way (non-blocking, faster).
 For an example how to create a client, a request and get the result see the example in the [sample folder][sample_folder].
 
+* [Age Mismatch](#get-age-mismatch-inference-information)
+* [Complete Order Discrepancy](#get-complete-order-discrepancy-inference-information)
+* [Critical Result](#get-critical-result-inference-information)
+* [Finding](#get-finding-information)
+* [Follow-up Communication](#get-follow-up-communication-information)
+* [Follow-up Recommendation](#get-follow-up-recommendation-information)
+* [Laterality Discrepancy](#get-laterality-discrepancy-information)
+* [Limited Order Discrepancy](#get-limited-order-discrepancy-information)
+* [Radiology Procedure](#get-radiology-procedure-information)
+* [Sex Mismatch](#get-sex-mismatch-information)
+
+### Running the samples
+
+1. Open a terminal window and `cd` to the directory that the samples are saved in.
+2. Set the environment variables specified in the sample file you wish to run.
+
+### Get Age Mismatch Inference information
```

Comment:
Before starting to go down the specific ways to access the inference results it would be nice to have an example showing how to make a request to the service.